Readme.md

```diff
@@ -40,6 +42,7 @@ brew 'emacs', args: ['with-cocoa', 'with-gnutls']
 cask 'google-chrome'
 cask 'java' unless system '/usr/libexec/java_home --failfast'
 cask 'firefox', args: { appdir: '~/my-apps/Applications' }
+mac_app_store '1Password', 443987910
```

Comment:
Can we call this DSL `app` or `mas`? `mac_app_store` is unnecessary long.

Also the number is confusing. Can we make it `app '1Password', id: 443987910`?